docs/glossary.md

```diff
@@ -0,0 +1,17 @@
+# Glossary
+
+**Playlist**: This is a representation of an HLS or DASH manifest.
+
+**Master Playlist Controller**: This acts as the main controller for the playback engine. It interacts with the SegmentLoaders, PlaylistLoaders, PlaybackWatcher, etc.
+
+**Playlist Loader**: This will request the source and load the master manifest. It also interacts with the ABR algorithm to pick a media playlist or wraps a media playlist if it is provided as the source. There are more details about the playlist loader [here](./arch.md).
+
+**Media Playlist**: This is a manifest that represents a single rendition of the source.
+
+**Segment Loader**: This controls the requesting and appending of segments on to the browser's [SourceBuffers](https://developer.mozilla.org/en-US/docs/Web/API/SourceBuffer).
+
+**ABR(Adaptive Bitrate) Algorithm**: This is defined in selectPlaylist and is described more [here](./bitrate-switching.md).
```

Comment:
ABR is a more generic term. We may want to point to https://en.wikipedia.org/wiki/Adaptive_bitrate_streaming and say that our "chosen" ABR algorithm is referenced by selectPlaylist (can also point to the README where it shows instructions for overriding it).